src/components/CurriculumList.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <section class="screen-box">
+    <Header
+      title="カリキュラム一覧"
+      url="/"
+    />
+    <main class="screen-box__main curriculum-list">
+      <div
+        v-for="(curriculum, index) in curriculumList"
```

Comment:
Please avoid using `index` as key. Each curriculum should has an unique key. The reason is explained in this link.
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Change to use the curriculum ID. Thanks.